src/main/resources/Diagram/DiagramEditSheet.xml

```diff
@@ -592,7 +592,15 @@ define('diagramStore', ['jquery', 'xwiki-meta', 'draw.io', 'xwiki-events-bridge'
       <cache>long</cache>
     </property>
     <property>
-      <code>define('diagramEditor', ['jquery', 'diagramStore'], function($, diagramStore) {
+      <code>define('diagramEditor', ['jquery', 'diagramStore', 'xwiki-meta'], function($, diagramStore, xm) {
+
+  /* These variables are used to decide if an image should be uploaded at original resolution or
+   * should be declined for being too big.
+   */
```

Comment:
The common practice AFAIK is to use the single line comment syntax ``//`` for (in-line) code comments and the multi-line comment syntax ``/**/`` for documenting functions, method, classes, etc.